updater/Gemfile.lock

```diff
@@ -27,6 +27,7 @@ PATH
       octokit (>= 4.6, < 7.0)
       parser (>= 2.5, < 4.0)
       psych (~> 5.0)
+      sorbet-runtime (~> 0.5)
```

Comment:
Why are these (only) in the _updater_ lockfile? 🤔 I'd expect the to be in the `common` lockfile instead?

Reply:
`Gemfile.lock` is explicitly ignored in `.gitignore` _except_ for in `updater`:

https://github.com/dependabot/dependabot-core/blob/ff4b17e9b2c742987a252099c26d182ca2408eda/.gitignore#L9-L11